From 0eda6fc529f3a1e3aebca4764fe9d772df694280 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Fri, 14 Dec 2018 22:24:14 +0100
Subject: [PATCH] hostapd: Add missing dependencies for eapol-test

Add missing dependency on libubox and libubus for eapol-test

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 package/network/services/hostapd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/services/hostapd/Makefile b/package/network/services/hostapd/Makefile
index a1480bf963..b21148f1f6 100644
--- a/package/network/services/hostapd/Makefile
+++ b/package/network/services/hostapd/Makefile
@@ -393,7 +393,7 @@ define Package/eapol-test
   SECTION:=net
   CATEGORY:=Network
   VARIANT:=supplicant-full-internal
-  DEPENDS:=$(DRV_DEPENDS)
+  DEPENDS:=$(DRV_DEPENDS) +libubox +libubus
 endef
 
 define Package/eapol-test-openssl
@@ -402,7 +402,7 @@ define Package/eapol-test-openssl
   CATEGORY:=Network
   VARIANT:=supplicant-full-openssl
   CONFLICTS:=$(filter-out eapol-test-openssl ,$(EAPOL_TEST_PROVIDERS))
-  DEPENDS:=$(DRV_DEPENDS) +libopenssl
+  DEPENDS:=$(DRV_DEPENDS) +libopenssl +libubox +libubus
   PROVIDES:=eapol-test
 endef
 
@@ -412,7 +412,7 @@ define Package/eapol-test-wolfssl
   CATEGORY:=Network
   VARIANT:=supplicant-full-wolfssl
   CONFLICTS:=$(filter-out eapol-test-openssl ,$(filter-out eapol-test-wolfssl ,$(EAPOL_TEST_PROVIDERS)))
-  DEPENDS:=$(DRV_DEPENDS) +libwolfssl
+  DEPENDS:=$(DRV_DEPENDS) +libwolfssl +libubox +libubus
   PROVIDES:=eapol-test
 endef
 
-- 
2.19.2

